lib/logger.js

```diff
@@ -1,27 +1,46 @@
 'use strict';
 
 const Winston = require('winston');
+const morgan = require('morgan');
+
+function Logger(level, filename) {
+  const logLevel = level || 'info';
 
-function Logger(config) {
   const logger = new Winston.Logger({
-    level: config.get('log:level'),
+    level: logLevel,
     transports: [
       new Winston.transports.Console({
         colorize: true,
         timestamp: true
       })
-    ]
+    ],
+    exitOnError: false
   });
 
-  const filename = config.get('log:filename');
-
   if (filename) {
-    logger.add(Winston.transports.File, {
-      filename
-    });
+    logger.add(Winston.transports.File, {filename});
```

Comment:
I think we're overcomplicating this. We should:

1. Remove all file-logging from the config/logging modules. If other users want to log to file, they should `> to_a_file`, or `| to_a_log_handler`. Systemd, upstart, runit, and daemontools all do this by default, or with minimal configuration.

2. Log API requests in the default log-stream. There will not be very many of these. Only local resources have access to the propsd API.